x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartTrainedModelDeploymentAction.java

```diff
@@ -141,6 +154,11 @@ public String getIndex() {
             return index;
         }
 
+        public long estimateMemoryUsageBytes() {
+            // While loading the model in the process we need twice the model size
```

Comment:
Should we also mention that we found through experiment that the in-memory size is approximately equal to the JSON size?  `modelBytes` is the JSON size right?

While it's intuitive that `in memory size = constant * JSON size`, discovering that `constant = 1` is an important piece of research and we don't want a future employee to have to redo it because we've lost track of the fact that the research has already been done.

This should probably also go in a Javadoc comment somewhere.